src/test/java/api/loans/scenarios/CheckInDeclaredLostItemTest.java

```diff
@@ -0,0 +1,477 @@
+
+public class CheckInDeclaredLostItemTest extends APITests {
+  private static final String LOST_ITEM_FEE = "Lost item fee";
+  private static final String LOST_ITEM_PROCESSING_FEE = "Lost item processing fee";
+  private static final String CANCELLED_ITEM_RETURNED = "Cancelled item returned";
+  private static final String REFUNDED_FULLY = "Refunded fully";
+  private static final String CREDITED_FULLY = "Credited fully";
+  private static final String LOST_ITEM_FOUND = "Lost item found";
+
+  private IndividualResource item;
+  private IndividualResource loan;
+
+  @Before
+  public void activateChargeableLostItemFeePolicy() {
+    useLostItemPolicy(lostItemFeePoliciesFixture.chargeFee().getId());
+  }
+
+  @Test
+  public void shouldCancelItemFeeOnlyWhenNoOtherFeesCharged() {
+    final double itemFee = 15.00;
+
+    useLostItemPolicy(lostItemFeePoliciesFixture.create(
+      lostItemFeePoliciesFixture.facultyStandardPolicy()
+        .withName("Test check in")
+        .doNotChargeProcessingFee()
+        .withSetCost(itemFee)).getId());
+
+    declareItemLost();
+
+    checkInFixture.checkInByBarcode(item);
+
+    verifyLostItemFeeAccount(isAccountClosed(itemFee));
+    verifyLostItemFeeAccountAction(isCloseActionCreated(itemFee));
+  }
+
+  @Test
+  public void shouldCancelItemProcessingFeeOnlyWhenNoOtherFeesCharged() {
+    final double processingFee = 12.99;
+
+    useLostItemPolicy(lostItemFeePoliciesFixture.create(
+      lostItemFeePoliciesFixture.facultyStandardPolicy()
+        .withName("Test check in")
+        .chargeProcessingFee()
+        .withLostItemProcessingFee(processingFee)
+        .withNoChargeAmountItem()).getId());
+
+    declareItemLost();
+
+    checkInFixture.checkInByBarcode(item);
+
+    verifyLostItemProcessingFeeAccount(isAccountClosed(processingFee));
+    verifyLostItemProcessingFeeAccountAction(isCloseActionCreated(processingFee));
+  }
+
+  @Test
+  public void shouldNotRefundProcessingFeeWhenPolicyStatesNotTo() {
+    final double processingFee = 12.99;
+
+    useLostItemPolicy(lostItemFeePoliciesFixture.create(
+      lostItemFeePoliciesFixture.facultyStandardPolicy()
+        .withName("Test check in")
+        .chargeProcessingFee()
+        .withLostItemProcessingFee(processingFee)
+        .doNotRefundProcessingFeeWhenReturned()
+        .withNoChargeAmountItem()).getId());
+
+    declareItemLost();
+
+    feeFineAccountFixture.pay(feeFineAccountFixture
+      .getLostItemProcessingFeeAccount(loan.getId()), processingFee);
+
+    checkInFixture.checkInByBarcode(item);
+
+    verifyLostItemProcessingFeeAccount(allOf(
+      hasJsonPath("amount", processingFee),
+      hasJsonPath("remaining", 0.0),
+      hasJsonPath("status.name", "Closed"),
+      hasJsonPath("paymentStatus.name", "Paid fully")));
+    verifyLostItemProcessingFeeAccountAction(
+      not(arePaymentRefundActionsCreated(processingFee)));
+  }
+
+  @Test
+  public void shouldNoRefundFeesWhenReturnedAfterRefundPeriod() {
```

Comment:
Oops, I think I had a typo in my suggestion

```suggestion
  public void shouldNotRefundFeesWhenReturnedAfterRefundPeriod() {
```